src/online2/online-ivector-feature.cc

```diff
@@ -225,7 +226,14 @@ void OnlineIvectorFeature::UpdateStatsForFrames(
         posterior[j].second *= info_.posterior_scale * weight;
     }
   }
-  lda_->GetFrames(frames, &feats);  // get features without CMN.
+
+  // Do we select features with online cmvn?
```

Comment:
this comment is confusing to me, it sounds like the person coding it was not sure.  better to remove.